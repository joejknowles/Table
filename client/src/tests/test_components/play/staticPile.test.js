import Component from '../../../components/play/staticPile';

import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('PlayablePile renders as before', () => {
  const component = shallowRenderer.render(<Component />);
  expect(component).toMatchSnapshot();
});
