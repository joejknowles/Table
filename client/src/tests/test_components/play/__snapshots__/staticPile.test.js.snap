exports[`test PlayablePile renders as before 1`] = `
<div
  className="pile">
  <div
    className="face-down playing-card" />
</div>
`;
